lib/export-task.js

```diff
@@ -96,6 +121,28 @@ async function insertNewTask() {
   });
 }
 
+async function getTasksThatCanBeRetried() {
```

Comment:
Add some documentation to the function (similar to the other functions in the file)